src/TLuaInterpreter.cpp

```diff
@@ -4310,17 +4230,13 @@ int TLuaInterpreter::setBackgroundColor(lua_State* L)
         r = static_cast<int>(lua_tonumber(L, s));
 
         if (!validRange(r)) {
-            lua_pushnil(L);
-            lua_pushfstring(L, "setBackgroundColor: bad argument #%d value (red value needs to be between 0-255, got %d!)", s, r);
-            return 2;
+            return warnArgumentValue(L, __func__, QStringLiteral("bad argument #%1 value (red value needs to be between 0-255, got %2!)").arg(s, r));
```

Comment:
Since you are now using `QString::arg(...)` you have got to either use a separate `.arg(int).arg(int)` form or convert them to text before one of the `QString::arg(...)` overloads thinks that the second number is a field width or number base or sommat - note that this applies to all cases where there is more than one argument and the second (and possibly higher) arguments are numbers but I am only pointing out this one!
```suggestion
            return warnArgumentValue(L, __func__, QStringLiteral("bad argument #%1 value (red value needs to be between 0-255, got %2!)").arg(QString::number(s), QString::number(r)));
```

Reply:
Good catch. Need to review!

Reply:
It's good for mixed argument types but not in general https://github.com/Mudlet/Mudlet/blob/development/.github/CONTRIBUTING.md#use-argarg1-arg2-instead-of-argarg1argarg2

Reply:
Then again, some of these are like (auto, auto) 😕

Reply:
Oh whoopsie, will switch to `QString::number(yourNumber)` then instead.
These lines now seem to get rather long and unwieldy. 
But I hesitate to split it into 2-3 every single time.